docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,31 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **Implemented**
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
+[Cookie Management](https://github.com/DevExpress/testcafe/issues/5964)                           | Planned
 
 ## API
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Configuration File](https://github.com/DevExpress/testcafe/issues/3131)                          | **Implemented**
-[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | **Implemented**
-[Browser Information in Test Code](https://github.com/DevExpress/testcafe/issues/481)             | **Implemented**
+[Dedicated *Scroll* Action](https://github.com/DevExpress/testcafe/issues/5965)                   | Planned
 
 ## Debugging
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Video Recording](https://github.com/DevExpress/testcafe/issues/2151)                             | **Implemented**
-[Live Mode](https://github.com/DevExpress/testcafe/issues/3215)                                   | **Implemented**
-[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[Interactive Demo Center](https://github.com/DevExpress/testcafe/issues/5966)                     | Planned
 
-## Ecosystem / Integrations
+## Test Stability
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | **Implemented**
-[Jenkins Plugin](https://github.com/DevExpress/testcafe/issues/4289)                              | **Implemented**
+[Retry Failed Requests for Tested Webpages](https://github.com/DevExpress/testcafe/pull/5738)     | **Implemented**
```

Comment:
```suggestion
[Automatically Retry Failed Requests](https://github.com/DevExpress/testcafe/pull/5738)     | **Implemented**
```

Reply:
Not all requests, just an initial request for the page (when we request HTML from a server)

Reply:
Your phrasing doesn't make this clear.

```suggestion
[Automatically Reissue Failed Page Requests](https://github.com/DevExpress/testcafe/pull/5738)     | **Implemented**
```

or 

```suggestion
[Repeat Page Requests on Failure](https://github.com/DevExpress/testcafe/pull/5738)     | **Implemented**
```